CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 # Braintree Android SDK Release Notes
 
 ## unreleased
+
+* PayPalNativeCheckout
+  * Bump native-checkout version to release `1.2.0`
+  * Adding in support for the merchant passed email feature (opt in only)
```

Comment:
Can we update this to something like "add `setUserAuthenticationEmail() to PayPalNativeRequest"?